src/main/java/core/basesyntax/Factorial.java

```diff
@@ -6,6 +6,11 @@
  */
 public class Factorial {
     public int getFactorial(int number) {
-        return 0;
+        int result = 1;
```

Comment:
can we solve this task without this variable?

Reply:
Done.